### PR TITLE
17 feat add capabilities to create apim tokens

### PR DESCRIPTION
**Pull request by ivica3730k**

### CODE REVIEW

**Comment by ivica3730k** [comment]
closes #24

File: app/main/controller/apim_controller.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+
+@api.route("/")
+class GetAPIMToken(Resource):
+    @api.doc(description="Get a APIM token")
+    @api.response(200, "Success")
+    @auth_decorator.header_decorator(
+        allowed_roles=["StacPortal.Viewer", "StacPortal.Creator"]
+    )
+    def get(self, oid: str, preferred_username: str, name: str):
```
Comment (james-hinton) [inline app/main/controller/apim_controller.py:19]: maybe im wrong here, but isnt for get its meant to come from the query args?

I guess it must work though?

**Comment by james-hinton** [comment]
LGTM

**Comment by ivica3730k** [comment]
Okay, thank you. Will merge it in once the terraform is in place with new environment variables

File: app/main/controller/apim_controller.py (hunk 1 above)
Comment (ivica3730k) [reply app/main/controller/apim_controller.py:19]: Sorry, havent seen it but replied to you in PM.

TLDR: It gets it from AD JWT
